utils/reporting/tfm/tfm-reporting.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * TFM REPORTING
+ * **************
+ * Purpose of this script is to generate a bespoke
+ * report of all completed deals from TFM
+ */
+
+const CONSTANTS = require('../../data-migration/constant');
+const { getCollection, disconnect } = require('../../data-migration/helpers/database');
+const { write } = require('../../data-migration/helpers/io');
+
+// ******************** DEALS *************************
+/**
+   * Return all the TFM deals with `MIA/MIN` filter.
+   * @param {Object} filter Mongo filter
+   * @returns {Object} Collection object
+   */
+const getTfmDeals = () => getCollection(
+  CONSTANTS.DATABASE.TABLES.TFM_DEAL,
+  { 'tfm.stage': 'Confirmed', 'tfm.dateReceivedTimestamp': { $gte: 1648684800 } },
```

Comment:
I wonder if `1648684800` can actually be a default value instead that can be overwritten if needed?
So the function can look something like this:
```
const getTfmDeals = (timestamp = 1648684800) => getCollection(...)
```

we can also add a comment that `1648684800` means  1 April 2022 in EPOCH (without milliseconds)

Reply:
Good suggestion, have amended.